docs/tutorials/get-started-with-ngo.md

```diff
@@ -116,7 +122,7 @@ You should see **Scenes/SampleScene** listed under **Scenes In Build**. You can
 
 :::tip
 
-Use the command line helper to test the build out before continuing. You should see the Player prefab spawn when:
+If you want, you can create a build at this point and use the [command line helper](command-line-helper) to test the build out before continuing. You should see the Player prefab spawn when:
```

Comment:
Is a build needed at this point as part of the guide? i.e. does the user need to make a build to validate previous steps? If not, we could skip this step.

Reply:
I was on the fence about this. I don't think it's necessary at this point in the guide, so I'll remove it.